brainzutils/cache.py

```diff
@@ -28,22 +32,37 @@
 ENCODING_ASCII = "ascii"
 
 
-def init(host="localhost", port=6379, db_number=0, namespace=""):
+def init(host: str = "localhost", port: int = 6379, db_number: int = 0,
+         namespace: str = "", client_name: Optional[str] = None):
```

Comment:
if you give a type a default value of None it implicitly makes it Optional, so Optional[str] isn't needed

Reply:
Fixed.